compute/compute_test.go

```diff
@@ -8,8 +8,9 @@ import (
 
 func TestCreateVM(t *testing.T) {
 	Convey("CreateVM", t, func() {
```

Comment:
Let us use one testing library - we have been using `gocheck` please use that.  I also going to add `prehooks` to reject code if it isn't "go fmt` ed.